.github/renovate.json5

```
@@ -0,0 +1,123 @@
+
+{
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+  enabled: true,
+  extends: [
+    'config:recommended',
+    ':dependencyDashboard',
+    ':gitSignOff',
+    'helpers:pinGitHubActionDigests',
+  ],
+  timezone: 'Europe/London',
+  labels: [
+    'dependencies',
+    'renovate',
+  ],
+  semanticCommits: 'enabled',
+  automerge: false,
+  pruneStaleBranches: true,
+  postUpgradeTasks: {
+    commands: [
+      "export PS1='> ' && make generate",
```

```suggestion
      'make generate',
```

This export appear very hackish. I would prefer trying to figure out why this is happening and solve the root cause instead.

Also suggesting consistent quote usage.

According to the Renovate docs, it appears like we also must set `allowedCommands`. Could you explain why this is working regardless?

---

The allowed commands is set globally in the github action as an ENV variable

---

Yes there is a fix for shells which are non interactive.. but I am not sure how to fix it on a repo by repo basis

---

I am absolutely not skilled in scripting. Something we can do in makefile-modules?

---

@hjoshi123 could you verify that this change works for you: https://github.com/cert-manager/makefile-modules/pull/324